src/seedu/addressbook/logic/Logic.java

```diff
@@ -85,7 +85,8 @@ public CommandResult execute(String userCommandText) throws Exception {
     private CommandResult execute(Command command) throws Exception {
```

Comment:
You need to update the header comment of this method as the behavior of _execute_ changes due to the way data is saved